### ISSUE TITLE
rst cause:4,boot mode(1,6)

### ISSUE THREAD

**Issue by sarwadenj** [esp8266/Arduino#2829]
problem coming after uploading program....
 
rst cause:4,boot mode(1,6)

GPIO's status after uploading code
CH_PD--VCC
GPIO 15--GND
GPIO 0-Connected to DC MOTOR via N-MOSFET
GPIO 2-Connected to interrupt pin for sensor.....

once MOTOR start rotating after 10sec it throws wdt reset......
previously it is not giving problem with this setup at normal load

**Comment by devyte** [comment]
@sarwadenj no sketch, no basic info, no IDE settings.
Closing due to lack of info, see #3655 . If the problem still exists, please open a new issue, and provide the details requested in the issue template.

**Comment by adavidzh** [comment]
I just started getting exactly the same thing after flashing:
```
1 30 03 03 22 a3 ff c0 22 01 30 03[...removed for brevity...]
 ets Jan  8 2013,rst cause:2, boot mode:(1,6)


 ets Jan  8 2013,rst cause:4, boot mode:(1,6)

wdt reset
```
It happens with any sketch, including
```
void setup() {
  // put your setup code here, to run once:
}
void loop() {
  // put your main code here, to run repeatedly:
}
```
I'm using the Arduino IDE 1.8.5, and `Generic ESP8266 Module` on a Ledunia. This Ledunia was working perfectly. I looked through #1013 without success.

If I reset by shorting `RES` to `GND`, I then get an infinite loop of
```
 ets Jan  8 2013,rst cause:2, boot mode:(3,6)

load 0x4010f000, len 1384, room 16 
tail 8
chksum 0x2d
csum 0x2d
v3de0c112
~ld
[...byte code...]
```
This seemed like #2414, but I am powering from the same place as when it was working (a USB port).

The whole thing started after uploading a sketch that was setting pin 13 as output. Could that be the reason? Any thoughts are welcome.

**Comment by Dynamite-7** [comment]
Hey everyone..
I'm getting this in serial monitor of arduino... please suggest some solutions, I'm unable to check my desired output.

Exception (9):
epc1=0x402027ae epc2=0x00000000 epc3=0x00000000 excvaddr=0x0000001f depc=0x00000000

ctx: cont
sp: 3ffefe40 end: 3fff0090 offset: 01a0

stack>>>
3ffeffe0: 0000004b 00000050 3ffeed48 402027a8
3ffefff0: 3ffe8cbc 3ffeefa0 3ffe8cb8 a0640534
3fff0000: 3ffe8994 3ffeefa0 0000004b 402032cc
3fff0010: 40104e5a 3ffeefa0 3ffeed48 402021cd
3fff0020: 00000000 00000000 00000000 402032a1
3fff0030: 3ffe8cbc 3ffeed6c 3fff15b4 0000003f
3fff0040: 00000038 3ffef060 3ffeefa0 402032cc
3fff0050: 3fff068c 40202578 3ffeefa0 3ffef060
3fff0060: 3fffdad0 00000000 3ffef058 402022c3
3fff0070: 3fffdad0 00000000 3ffef058 402038a0
3fff0080: feefeffe feefeffe 3ffef070 40100710
<<<stack<<<

ets Jan 8 2013,rst cause:2, boot mode:(1,6)

ets Jan 8 2013,rst cause:4, boot mode:(1,6)

wdt reset

Please help ASAP..!!!

Thanks.

**Comment by adavidzh** [comment]
For some reason, after a few weeks of not touching the board, I no longer have the problem.

**Comment by dangsonbk** [comment]
I got same error when my code looped in while(1);. Seem that the board was reset by watchdog.

**Comment by not4unco** [comment]
I'm having the same issue with my NodeMCU...

09:48:45.503 -> Sdk version: 3.0.0-dev(c0f7b44)
09:48:45.503 -> Core Version: 2_5_0
09:48:45.503 -> Boot Version: 31
09:48:45.503 -> Boot Mode: 1
09:48:45.503 -> CPU Frequency: 80 MHz

```
11:19:04.627 -> Soft WDT reset
11:19:04.627 -> 
11:19:04.627 -> >>>stack>>>
11:19:04.627 -> 
11:19:04.627 -> ctx: sys
11:19:04.627 -> sp: 3fffed40 end: 3fffffb0 offset: 01b0
11:19:04.627 -> 3fffeef0:  3ffe8d00 64e525a5 60000600 60000600  
11:19:04.627 -> 3fffef00:  00000000 3ffed318 00000000 00000000  
11:19:04.627 -> 3fffef10:  4022d51c 3ffed318 3ffee2d0 60000600  
11:19:04.627 -> 3fffef20:  4022fe65 3ffed318 3ffee2d0 447e425a  
11:19:04.627 -> 3fffef30:  4022feaa 3fffdab0 00000000 3fffdcb0  
11:19:04.906 -> 3fffef40:  3ffee2f8 3fffdab0 00000000 4020644b  
11:19:04.906 -> 3fffef50:  40000f49 40000f49 3fffdab0 40000f49  
11:19:04.906 -> 3fffef60:  40000e19 40001878 00000002 00000000  
11:19:04.906 -> 3fffef70:  3fffff10 ffffffff ffffffff 40105694  
11:19:04.906 -> 3fffef80:  4010569a 00000002 00000000 173243cc  
11:19:04.906 -> 3fffef90:  4010000d 5797346d 62d64d98 f57879d7  
11:19:04.906 -> 3fffefa0:  40101868 3fffef3c 40101815 3fffff48  
11:19:04.906 -> 3fffefb0:  3fffffc0 00000000 00000000 feefeffe  
11:19:04.906 -> 3fffefc0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3fffefd0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3fffefe0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3fffeff0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff000:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff010:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff020:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff030:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff040:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff050:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff060:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff070:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff080:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff090:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff0a0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff0b0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff0c0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff0d0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff0e0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff0f0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff100:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff110:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff120:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff130:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff140:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff150:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff160:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff170:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff180:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff190:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff1a0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff1b0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff1c0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff1d0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff1e0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff1f0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff200:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff210:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff220:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff230:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff240:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff250:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff260:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff270:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff280:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff290:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff2a0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff2b0:  feefeffe feefeffe feefeffe feefeffe  
11:19:04.906 -> 3ffff2c0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff2d0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff2e0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff2f0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff300:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff310:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff320:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff330:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff340:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff350:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff360:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff370:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff380:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff390:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff3a0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff3b0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff3c0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff3d0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff3e0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff3f0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff400:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff410:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff420:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff430:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff440:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff450:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff460:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff470:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff480:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff490:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff4a0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff4b0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff4c0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff4d0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.061 -> 3ffff4e0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.202 -> 3ffff4f0:  feefeffe feefeffe feefeffe feefeffe  
11:19:05.202 -> 3ffff500:  feefeffe feefeffe feefeffe 3ffeee14  
11:19:05.202 -> 3ffff510:  00000010 00000010 00000000 4010166d  
11:19:05.202 -> 3ffff520:  feefeffe 40206137 feefeffe 3ffff705  
11:19:05.202 -> 3ffff530:  3fffff14 3ffffca0 3fffff10 40202f47  
11:19:05.202 -> 3ffff540:  00000001 3ffff70c feefeffe 3ffffb00  
11:19:05.202 -> 3ffff550:  00000010 3fffff10 3fffff10 40206137  
11:19:05.202 -> 3ffff560:  3ffff620 00000004 3fffff10 40206183  
11:19:05.202 -> 3ffff570:  00000000 3ffff5a0 3fffff10 402061cc  
11:19:05.202 -> 3ffff580:  3ffe87fc 3fffff10 3fffff10 40206202  
11:19:05.202 -> 3ffff590:  3ffeee14 00000154 00000154 4010138c  
11:19:05.202 -> 3ffff5a0:  3ffe87fc 3fffff00 3fffff38 401017d0  
11:19:05.202 -> 3ffff5b0:  00000000 feefeffe 3ffee948 402060f4  
11:19:05.202 -> 3ffff5c0:  3ffe87fc 3fffff00 447f8000 40206110  
11:19:05.202 -> 3ffff5d0:  3ffeeb48 3fffff00 447f8000 40203c80  
11:19:05.202 -> 3ffff5e0:  726f6f63 6f6c0064 382d006e 36312e34  
11:19:05.202 -> 3ffff5f0:  74616c00 2e313300 77003835 68746165  
11:19:05.202 -> 3ffff600:  69007265 32370064 616d0031 48006e69  
11:19:05.202 -> 3ffff610:  00657a61 63736564 74706972 006e6f69  
11:19:05.202 -> 3ffff620:  657a6168 6f636900 3035006e 61620064  
11:19:05.202 -> 3ffff630:  73006573 69746174 00736e6f 6e69616d  
11:19:05.202 -> 3ffff640:  6d657400 37310070 0039352e 73657270  
11:19:05.202 -> 3ffff650:  65727573 32303100 75680032 6964696d  
11:19:05.202 -> 3ffff660:  37007974 65740032 6d5f706d 31006e69  
11:19:05.202 -> 3ffff670:  00312e36 706d6574 78616d5f 2e383100  
11:19:05.202 -> 3ffff680:  69760038 69626973 7974696c 30363100  
11:19:05.202 -> 3ffff690:  77003339 00646e69 65657073 2e320064  
11:19:05.202 -> 3ffff6a0:  65640036 36310067 6c630030 7364756f  
11:19:05.202 -> 3ffff6b0:  6c6c6100 64003100 35310074 35353934  
11:19:05.202 -> 3ffff6c0:  30303331 73797300 70797400 00310065  
11:19:05.202 -> 3ffff6d0:  36006469 00373230 7373656d 00656761  
11:19:05.202 -> 3ffff6e0:  30302e30 63003234 746e756f 55007972  
11:19:05.202 -> 3ffff6f0:  75730053 7369726e 35310065 34353934  
11:19:05.563 -> 3ffff700:  33393232 6e757300 00746573 39343531  
11:19:05.563 -> 3ffff710:  34313835 69003532 31340064 32333937  
11:19:05.563 -> 3ffff720:  616e0030 4100656d 6e61626c 6f630079  
11:19:05.563 -> 3ffff730:  30320064 72740030 223a2279 2c225355  
11:19:05.563 -> 3ffff740:  6e757322 65736972 35313a22 34353934  
11:19:05.563 -> 3ffff750:  33393232 7573222c 7465736e 35313a22  
11:19:05.563 -> 3ffff760:  38353934 35323431 69222c7d 343a2264  
11:19:05.796 -> 3ffff770:  33393731 222c3032 656d616e 41223a22  
11:19:05.796 -> 3ffff780:  6e61626c 222c2279 22646f63 3030323a  
11:19:05.796 -> 3ffff790:  63227b7d 64726f6f 227b3a22 226e6f6c  
11:19:05.796 -> 3ffff7a0:  34382d3a 2c36312e 74616c22 31333a22  
11:19:05.796 -> 3ffff7b0:  7d38352e 6577222c 65687461 203a2272  
11:19:05.796 -> 3ffff7c0:  6469227b 30383a22 6d222c30 226e6961  
11:19:05.796 -> 3ffff7d0:  6c43223a 22726165 6564222c 69726373  
11:19:05.796 -> 3ffff7e0:  6f697470 223a226e 61656c63 6b732072  
11:19:05.796 -> 3ffff7f0:  222c2279 6e6f6369 30223a22 7d226431  
11:19:05.796 -> 3ffff800:  62222c20 22657361 7473223a 6f697461  
11:19:05.796 -> 3ffff810:  2c22736e 69616d22 7b3a226e 6d657422  
11:19:05.796 -> 3ffff820:  313a2270 35362e39 7270222c 75737365  
11:19:05.796 -> 3ffff830:  3a226572 32323031 7568222c 6964696d  
11:19:05.796 -> 3ffff840:  3a227974 222c3536 706d6574 6e696d5f  
11:19:05.796 -> 3ffff850:  38313a22 6574222c 6d5f706d 3a227861  
11:19:05.796 -> 3ffff860:  2c7d3132 73697622 6c696269 22797469  
11:19:05.796 -> 3ffff870:  3036313a 222c3339 646e6977 227b3a22  
11:19:05.796 -> 3ffff880:  65657073 333a2264 222c312e 22676564  
11:19:05.796 -> 3ffff890:  3039313a 63222c7d 64756f6c 7b3a2273  
11:19:05.796 -> 3ffff8a0:  6c6c6122 7d313a22 7464222c 35313a22  
11:19:05.796 -> 3ffff8b0:  35353934 30303532 7973222c 7b3a2273  
11:19:05.796 -> 3ffff8c0:  70797422 313a2265 6469222c 32333a22  
11:19:05.796 -> 3ffff8d0:  222c3830 7373656d 22656761 302e303a  
11:19:05.796 -> 3ffff8e0:  2c373330 756f6322 7972746e 55223a22  
11:19:05.796 -> 3ffff8f0:  222c2253 726e7573 22657369 3435313a  
11:19:05.796 -> 3ffff900:  32343539 2c323932 6e757322 22746573  
11:19:05.796 -> 3ffff910:  3435313a 31383539 7d363234 6469222c  
11:19:05.796 -> 3ffff920:  31343a22 32333937 6e222c30 22656d61  
11:19:05.796 -> 3ffff930:  6c41223a 796e6162 63222c22 3a22646f  
11:19:05.796 -> 3ffff940:  7d303032 6f63227b 2264726f 6c227b3a  
11:19:05.796 -> 3ffff950:  3a226e6f 2e34382d 222c3631 2274616c  
11:19:05.796 -> 3ffff960:  2e31333a 2c7d3835 61657722 72656874  
11:19:05.796 -> 3ffff970:  7b203a22 22646922 3030383a 616d222c  
11:19:05.796 -> 3ffff980:  3a226e69 656c4322 2c227261 73656422  
11:19:05.796 -> 3ffff990:  70697263 6e6f6974 63223a22 7261656c  
11:19:05.796 -> 3ffff9a0:  796b7320 69222c22 226e6f63 3130223a  
11:19:05.796 -> 3ffff9b0:  207d2264 6162222c 3a226573 61747322  
11:19:05.796 -> 3ffff9c0:  6e6f6974 222c2273 6e69616d 227b3a22  
11:19:05.796 -> 3ffff9d0:  706d6574 31323a22 2c34382e 65727022  
11:19:05.796 -> 3ffff9e0:  72757373 313a2265 2c323230 6d756822  
11:19:05.796 -> 3ffff9f0:  74696469 353a2279 74222c33 5f706d65  
11:19:05.796 -> 3ffffa00:  226e696d 2c31323a 6d657422 616d5f70  
11:19:05.796 -> 3ffffa10:  323a2278 222c7d33 69736976 696c6962  
11:19:05.796 -> 3ffffa20:  3a227974 39303631 77222c33 22646e69  
11:19:05.796 -> 3ffffa30:  73227b3a 64656570 2e343a22 64222c31  
11:19:05.796 -> 3ffffa40:  3a226765 7d303831 6c63222c 7364756f  
11:19:05.796 -> 3ffffa50:  227b3a22 226c6c61 2c7d313a 22746422  
11:19:05.796 -> 3ffffa60:  3435313a 34353539 2c303039 73797322  
11:19:05.796 -> 3ffffa70:  227b3a22 65707974 2c313a22 22646922  
11:19:05.796 -> 3ffffa80:  3032333a 6d222c38 61737365 3a226567  
11:19:05.796 -> 3ffffa90:  30302e30 222c3334 6e756f63 22797274  
11:19:05.796 -> 3ffffaa0:  5355223a 73222c22 69726e75 3a226573  
11:19:05.796 -> 3ffffab0:  39343531 32323435 222c3139 736e7573  
11:19:05.796 -> 3ffffac0:  3a227465 39343531 34313835 2c7d3832  
11:19:05.796 -> 3ffffad0:  22646922 3731343a 30323339 616e222c  
11:19:05.796 -> 3ffffae0:  3a22656d 626c4122 22796e61 6f63222c  
11:19:05.796 -> 3ffffaf0:  323a2264 ff7d3030 3ffe0000 40203cb6  
11:19:05.796 -> 3ffffb00:  40206ba8 3ffffb10 00000400 00000238  
11:19:05.796 -> 3ffffb10:  3ffffb00 3ffffb40 3ffffb00 3ffffb20  
11:19:05.796 -> 3ffffb20:  3ffffb30 3ffff5e6 00000001 3ffff5ea  
11:19:05.796 -> 3ffffb30:  00000000 3ffff5f1 00000001 3ffff5f5  
11:19:05.796 -> 3ffffb40:  3ffffb98 3ffff5e0 00000007 3ffffb18  
11:19:05.796 -> 3ffffb50:  3ffffb00 3ffffb58 3ffffb68 3ffff603  
11:19:05.796 -> 3ffffb60:  00000001 3ffff606 3ffffb78 3ffff60a  
11:19:05.796 -> 3ffffb70:  00000002 3ffff60f 3ffffb88 3ffff614  
11:19:05.796 -> 3ffffb80:  00000002 3ffff620 00000000 3ffff625  
11:19:05.796 -> 3ffffb90:  00000002 3ffff62a 3ffffba8 3ffff5fb  
11:19:05.796 -> 3ffffba0:  00000007 3ffffb50 3ffffc10 3ffff62e  
11:19:05.796 -> 3ffffbb0:  00000002 3ffff633 3ffffb00 3ffffbc0  
11:19:05.796 -> 3ffffbc0:  3ffffbd0 3ffff641 00000001 3ffff646  
11:19:05.796 -> 3ffffbd0:  00000001 00000004 40100e6b 00000000  
11:19:05.796 -> 3ffffbe0:  00000000 00000000 00000000 00000004  
11:19:05.796 -> 3ffffbf0:  40100e6b 00000000 00000000 00000000  
11:19:05.796 -> 3ffffc00:  00000000 4000050c 00000000 00000000  
11:19:05.796 -> 3ffffc10:  40100e6b 00000030 0000001c ffffffe0  
11:19:05.917 -> 3ffffc20:  00000001 00000004 40100e6b 00000000  
11:19:05.917 -> 3ffffc30:  00000000 00000000 00000000 00000004  
11:19:05.917 -> 3ffffc40:  40100e6b 00000000 00000000 00000000  
11:19:05.917 -> 3ffffc50:  00000000 4000050c 00000000 40100e68  
11:19:05.917 -> 3ffffc60:  40100e6b 00000030 0000001c ffffffe0  
11:19:05.917 -> 3ffffc70:  40100e68 60000324 00000010 00000019  
11:19:05.917 -> 3ffffc80:  00000019 00000005 00000000 003a5400  
11:19:05.917 -> 3ffffc90:  000000c3 3fff0118 fff40000 00000000  
11:19:05.917 -> 3ffffca0:  00000004 ffffffff 00000020 00000030  
11:19:05.917 -> 3ffffcb0:  c0017035 000002b2 ff000000 3ffee900  
11:19:05.917 -> 3ffffcc0:  3ffe8fc5 4010558b 3ffec9d8 00000020  
11:19:05.917 -> 3ffffcd0:  3ffe8fc5 4010558b 3ffeca00 00000030  
11:19:05.917 -> 3ffffce0:  40102ea1 3ffeca00 3ffed474 40103072  
11:19:05.917 -> 3ffffcf0:  fffffff5 44772342 3ffed474 40103072  
11:19:05.917 -> 3ffffd00:  6a0e805a 00000000 00000000 401001eb  
11:19:05.917 -> 3ffffd10:  6a0d45f8 44772342 401034b2 00000020  
11:19:05.917 -> 3ffffd20:  ffffffff 00000004 00000000 401001eb  
11:19:05.917 -> 3ffffd30:  40100e79 00000004 00000000 40100e68  
11:19:05.917 -> 3ffffd40:  c0017005 00000004 00000000 40100e68  
11:19:05.917 -> 3ffffd50:  c0017005 00000004 00000000 00000022  
11:19:05.917 -> 3ffffd60:  3fffc200 40100da8 3fffc258 4000050c  
11:19:05.917 -> 3ffffd70:  4010032b 00000030 0000001c ffffffff  
11:19:05.917 -> 3ffffd80:  402015b4 00000015 00000000 60000318  
11:19:05.917 -> 3ffffd90:  c0017035 000002b2 00000040 003a5400  
11:19:05.917 -> 3ffffda0:  000000c3 3fff0118 fff40000 00000020  
11:19:05.917 -> 3ffffdb0:  00000010 3ffe8563 60000314 00000030  
11:19:05.917 -> 3ffffdc0:  3ffe8563 3ffee9e4 3ffee9e3 00000030  
11:19:05.917 -> 3ffffdd0:  00000010 3ffe8563 3ffee9e3 00000030  
11:19:05.917 -> 3ffffde0:  00000010 3ffe8563 3ffee9e3 00000030  
11:19:05.917 -> 3ffffdf0:  00000010 3ffe8563 3ffee9e3 00000030  
11:19:05.917 -> 3ffffe00:  00000010 3ffe8563 3ffee9e3 00000030  
11:19:05.917 -> 3ffffe10:  00000001 3ffee928 00000001 40204f42  
11:19:05.917 -> 3ffffe20:  00000001 00000001 00000003 40100d15  
11:19:05.917 -> 3ffffe30:  3ffee9e4 00000000 3ffee8e8 402054ec  
11:19:05.917 -> 3ffffe40:  00000001 00000001 00000003 40100d15  
11:19:05.917 -> 3ffffe50:  3ffee9e4 00000000 3ffee8e8 3ffee9e3  
11:19:05.917 -> 3ffffe60:  3ffee9e4 00000000 00000000 40201625  
11:19:05.917 -> 3ffffe70:  3ffee9e4 00000001 3ffee9e4 3ffee9e3  
11:19:05.917 -> 3ffffe80:  3ffee9e4 00000000 00000000 40201625  
11:19:05.917 -> 3ffffe90:  3ffee9e4 00000001 3ffee9e4 4020180d  
11:19:05.917 -> 3ffffea0:  00000027 00000001 3ffee9e4 4020180d  
11:19:05.917 -> 3ffffeb0:  00000027 00000001 3ffee9e4 3fff0118  
11:19:05.917 -> 3ffffec0:  00000051 3ffee928 3ffee9e1 40205390  
11:19:05.917 -> 3ffffed0:  00000059 3ffee928 3ffee9d0 402053b8  
11:19:05.917 -> 3ffffee0:  40100cec 00000051 3ffee928 40204ee5  
11:19:05.917 -> 3ffffef0:  00000001 3ffee928 00000051 40204f42  
11:19:05.917 -> 3fffff00:  00000001 3ffee928 00000051 40204f6c  
11:19:05.917 -> 3fffff10:  00000000 00000650 3ffee928 40204fa6  
11:19:05.917 -> 3fffff20:  3ffee928 00000003 3fff0117 40204fc3  
11:19:05.917 -> 3fffff30:  40105fb0 64fb6275 3ffeed2c 00000000  
11:19:05.917 -> 3fffff40:  3ffee2d0 3ffeed2c 3ffe8590 3ffeed2c  
11:19:05.917 -> 3fffff50:  3fffdad0 3ffeec54 40206490 3fffefa0  
11:19:05.917 -> 3fffff60:  3ffeed2c 3fffdad0 00001388 402019f7  
11:19:05.917 -> 3fffff70:  3fffdad0 00001388 3ffee93c 40203d93  
11:19:05.917 -> 3fffff80:  00000000 00000000 00000000 00000000  
11:19:05.917 -> 3fffff90:  00000000 00000000 00000001 402064b1  
11:19:05.917 -> 3fffffa0:  3fffdad0 00000000 3ffeec24 4020653c  
11:19:05.917 -> <<<stack<<<
11:19:05.917 -> 
11:19:05.917 ->  ets Jan  8 2013,rst cause:2, boot mode:(1,6)
11:19:05.917 -> 
11:19:10.020 -> 
11:19:10.020 ->  ets Jan  8 2013,rst cause:4, boot mode:(1,6)
11:19:10.020 -> 
11:19:10.020 -> wdt reset
```

**Comment by SkomisLTU** [comment]
cause:4, boot mode:(1,6)

I got this when used DS18B20 sensor with U8g2lib.h display. :(

**Comment by faelpinho** [comment]
Have no solution?
I got only this message when GPIO_0 is GND:
" ets Jan  8 2013,rst cause:2, boot mode:(3,6)

ets_main.c "

Nothing more. 
And I got this message when GPIO_0 is VCC:
" ets Jan  8 2013,rst cause:2, boot mode:(1,7)"
Nothing more.

I have clicked "Erase" button on ESP8266 Download Tool V.3.6.8, and the prompt shows this message:
CONNECT BAUD: 115200
==========
.Uploading stub...
Running stub...
Stub running...
------------------
crc_efuse_4bit: 0
crc_calc_4bit: 8
48bit mac
"

![Sem título](https://user-images.githubusercontent.com/1687631/71702321-7c867480-2dad-11ea-9792-16eaf8f7e1d2.png)
